src/components/EditableHeading/__tests__/EditableHeading-interactions.tsx

```diff
@@ -0,0 +1,73 @@
+import { expect } from "@storybook/jest";
+import {
+  Canvas,
+  clearText,
+  clickElement,
+  getByTestId,
+  getByRole,
+  interactionSuite,
+  typeText
+} from "../../../tests/interactions-utils";
+
+import { ComponentDefaultTestId } from "../../../tests/constants";
+import { resetFocus } from "../../../__tests__/interactions-helper";
+
+const text = "This heading is an editable heading";
+
+function getComponent(canvas: Canvas) {
+  return getByTestId(canvas, ComponentDefaultTestId.CLICKABLE);
+}
+function getHeading(canvas: Canvas) {
+  return getByTestId(canvas, ComponentDefaultTestId.CLICKABLE);
+}
+
+function getInput(canvas: Canvas) {
+  return getByRole(canvas, "input");
+}
+
+async function testBasicEditing(canvas: Canvas) {
+  const compponent = getComponent(canvas);
+  clickElement(compponent);
+
+  const input = getInput(canvas);
+  await clearText(input);
+  await typeText(input, text);
+  expect(input).toHaveAttribute("value", text);
+
+  await resetFocus();
+  const heading = getHeading(canvas);
+  expect(heading).toHaveTextContent(text);
```

Comment:
worth splitting to another test, just so chromatic will catch both

Reply:
What do you mean? I want to check that the text is changing when you blur after you change it, and the next test also checks the case where you just clear the text

Reply:
So the first text will be the blur only. The second test would be a component which already has a value set and then clear it